packages/p2p/src/hooks/__tests__/use-p2p-advertiser-adverts.spec.ts

```diff
@@ -0,0 +1,136 @@
+        expect(adverts_list?.[0]?.advertiser_details?.name).toBe('client Test90000253');
+    });
+    it('should return the advertiser adverts object from response when is_advertiser_info_subscribed is false and counterparty_advertiser_id is defined', () => {
+        mockUseStores.mockReturnValueOnce({
+            ...mock_return_value,
+        });
+        const { result } = renderHook(() => useP2PAdvertiserAdverts());
+        const adverts_list = result.current.adverts;
+
+        expect(adverts_list).toHaveLength(1);
+        expect(adverts_list?.[0]?.country).toBe('id');
+        expect(adverts_list?.[0]?.counterparty_type).toBe('buy');
+        expect(adverts_list?.[0]?.description).toBe('Created by script. Please call me 02203400');
+        expect(adverts_list?.[0]?.advertiser_details?.name).toBe('client Test90000253');
+    });
+
+    it('should return the advertiser adverts object from response when is_advertiser_info_subscribed is false and counterparty_advertiser_id is undefined', () => {
+        mockUseStores.mockReturnValueOnce({
+            ...mock_return_value,
+            general_store: {
+                ...mock_return_value.general_store,
+                counterparty_advertiser_id: undefined,
+            },
+        });
+        const { result } = renderHook(() => useP2PAdvertiserAdverts());
+        const adverts_list = result.current.adverts;
+
+        expect(adverts_list).toHaveLength(1);
+        expect(adverts_list?.[0]?.country).toBe('id');
+        expect(adverts_list?.[0]?.counterparty_type).toBe('buy');
+        expect(adverts_list?.[0]?.description).toBe('Created by script. Please call me 02203400');
+        expect(adverts_list?.[0]?.advertiser_details?.name).toBe('client Test90000253');
+    });
+    it('should return the advertiser adverts object from response when is_advertiser_info_subscribed is true and counterparty_advertiser_id is undefined', () => {
+        mockUseStores.mockReturnValueOnce({
+            ...mock_return_value,
+            general_store: {
+                ...mock_return_value.general_store,
+                is_advertiser_info_subscribed: true,
+                counterparty_advertiser_id: undefined,
+            },
+        });
+        const { result } = renderHook(() => useP2PAdvertiserAdverts());
+        const adverts_list = result.current.adverts;
+
+        expect(adverts_list).toHaveLength(1);
+        expect(adverts_list?.[0]?.country).toBe('id');
+        expect(adverts_list?.[0]?.counterparty_type).toBe('buy');
+        expect(adverts_list?.[0]?.description).toBe('Created by script. Please call me 02203400');
+        expect(adverts_list?.[0]?.advertiser_details?.name).toBe('client Test90000253');
+    });
+
+    it('should return the advertiser adverts object from response when selected_local_currency is truthy', () => {
+        mockUseStores.mockReturnValueOnce({
+            ...mock_return_value,
+        });
```

Comment:
```suggestion
        mockUseStores.mockReturnValueOnce(mock_return_value);
```
nit-pick ) the rest LGTM 👍🏻